galley/pkg/runtime/projections/serviceentry/handler.go

```diff
@@ -121,9 +124,12 @@ func (h *Handler) Handle(event resource.Event) {
 	case metadata.K8sCoreV1Services.Collection:
 		// Update the projections
 		h.handleServiceEvent(event)
-	case metadata.K8sCoreV1Nodes.Collection, metadata.K8sCoreV1Pods.Collection:
-		// update the pod cache.
-		h.podAndNodeHandler.Handle(event)
+	case metadata.K8sCoreV1Nodes.Collection:
+		// Update the pod cache.
```

Comment:
%s/pod/node ?

Reply:
it's actually the pod cache in both cases. I used to have separate node/pod caches, but now they're all merged into the pod cache.